### ISSUE TITLE
nemo_nlp.utils not found

### ISSUE THREAD

**Issue by vr25** [NVIDIA/NeMo#84]
Hi,

After following the instructions for [pre-training](https://nvidia.github.io/NeMo/nlp/pretraining.html) and [installation](https://nvidia.github.io/NeMo/installation.html), I get the following error:

nemo_nlp.utils not found

There are following two cases:

1. Installing nemo_nlp with pip install nemo_nlp doesn't create utils directory in "lib/python3.6/site-packages"

2. Installing nemo_nlp by cloning the git link and then running setup.py creates the utils directory but it still says "nemo_nlp.utils not found"

Thanks!

**Comment by okuchaiev** [comment]
Could you please try the latest master branch version and see if problem persists?
IMPORTANT: please uninstall all previous versions of: "nemo", "nemo_toolkit", "nemo_nlp"

**Comment by vr25** [comment]
Hi,

Thank you for your prompt response. I uninstalled "nemo", "nemo_toolkit", "nemo_nlp" and cloned the latest master repo. It throws the following error:
![image](https://user-images.githubusercontent.com/22553367/67995406-a80ba800-fc20-11e9-812a-0a6d8ffedfc5.png)

I then installed "nemo", "nemo_toolkit", "nemo_nlp" but same error:
![image](https://user-images.githubusercontent.com/22553367/67995394-92967e00-fc20-11e9-8ff9-0cb7042c88a0.png)


Thanks!
